recipes/norm/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+from conans import ConanFile, CMake, tools
+import os
+import glob
+import yaml
+
+
+class NormConan(ConanFile):
+    name = "norm"
+    description = "A reliable multicast transport protocol"
+    topics = ("conan", "norm", "multicast", "transport protocol")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.nrl.navy.mil/itd/ncs/products/norm"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    license = "NRL"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder)
```

Comment:
I believe that `destination` in `tools.get()` is very new, which conan version?

Reply:
1.22, will add requirement

Reply:
Added

Reply:
Actually, I've checked in conan client source code, and it was already there in 1.0.0. I don't understand why we don't always use it in CCI recipes.

Reply:
Cool, guess the docs were not updated yet, only looked there :)